tempesta_fw/http_parser.c

```diff
@@ -67,7 +67,11 @@
 	tfw_http_msg_set_str_data(msg, field, pos)
 
 #define __msg_field_fixup(field, pos)					\
-	tfw_http_msg_add_str_data(msg, field, data, __data_off(pos))
+do {									\
+	if (tfw_http_msg_add_str_data(msg, field, data,			\
```

Comment:
I think it makes sense to use `unliekly` in the conditions.

What happens if we hit the allocation error in any of the cases? Will we see an error message in the log or the HTTP message is just silently dropped?

Reply:
We get a single message from `WARN_ON_ONCE` at mm/page_alloc.c:3877 (__alloc_pages_slowpath), and a couple of warnings from Tempesta:
```
[ 5890.438319] [tempesta fw] Warning: Cannot grow HTTP data string
[ 5890.523323] [tempesta fw] Warning: response dropped: processing error: 192.168.122.1
```
Client receives a `HTTP/1.1 502 Bad Gateway` message.

Reply:
> it makes sense to use unlikely in the conditions

Completely forgot about them. Added. Thanks!